alacritty.yml

```diff
@@ -78,6 +78,12 @@
   # Set this to `None` to use the default theme variant.
   #gtk_theme_variant: None
 
+  # Disable vsync (changes require restart)
+  #
+  # Force to disable vsync, what can improve the input latency but can also add tearing.
+  # On Wayland, vsync is always disabled, setting this option has no effect.
+  #disable_vsync: false
```

Comment:
I think having this option as just `vsync` would make more sense.

Reply:
I also wanted to call it `vsync`, but it was a bit misleading, because on Wayland for example, even if `vsync` is set to `true`, it's not gonna be forced.
That's why I called it `disable_vsync`.